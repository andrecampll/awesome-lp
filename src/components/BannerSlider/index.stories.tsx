import { Story, Meta } from '@storybook/react/types-6-0';
import BannerSlider from './index';

export default {
  title: 'BannerSlider',
  component: BannerSlider,
} as Meta;

export const Basic: Story = () => <BannerSlider />;
