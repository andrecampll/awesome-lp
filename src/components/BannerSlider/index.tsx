import { Wrapper } from './styles';

const BannerSlider = () => (
  <Wrapper>
    <h1>BannerSlider</h1>
  </Wrapper>
);

export default BannerSlider;
