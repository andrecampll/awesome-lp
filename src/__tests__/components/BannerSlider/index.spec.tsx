import { render, screen } from '@testing-library/react';
import BannerSlider from '../../../components/BannerSlider';

describe('BannerSlider', () => {
  it('should be able to render', () => {
    const { container } = render(<BannerSlider />);

    expect(
      screen.getByRole('heading', {
        name: /BannerSlider/i,
      }),
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
