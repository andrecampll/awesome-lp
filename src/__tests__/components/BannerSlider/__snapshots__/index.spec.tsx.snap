// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerSlider should be able to render 1`] = `
<div
  class="sc-bdfBwQ"
>
  <h1>
    BannerSlider
  </h1>
</div>
`;
